### PR TITLE
Update content

### PR DESCRIPTION
**Pull request by AngeloCG97**
### What does this PR have?
1. Change icons color
1. Fix size aditional resource component 
1. Refactor: blog improvements
2. "do-you-need-blockchain" tool improvements
3. Add new section in teams page

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,3 +20,5 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 yarn.lock
+
+
```
Comment (xavier506) [inline .gitignore:24]: trailing space

File: blog/2021-03-09-Do-you-need-blockchain.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## Contact Us for More Information About Blockchain
+
+EOS Costa Rica operates since 2018. Our team develops blockchain-based solutions with great attention to detail in creating user-centered dapps for enterprise use. We also provide resources to the EOSIO infrastructure and promote the local EOSIO ecosystem. Let’s talk about how you can implement this technology into your organization. 
+
+[Contact Us](https://eoscostarica.io/contact-us)
+
+** Follow us on social media:**
+
+*   Website: <https://eoscostarica.io/home/>
```
Comment (xavier506) [inline blog/2021-03-09-Do-you-need-blockchain.mdx:42]: Is GitHub something we want to include on this list?

**Review by xavier506 — APPROVED** [review]
LGTM

File: package.json [hunk 3]
```diff
@@ -5,24 +5,28 @@
   "scripts": {
     "docusaurus": "docusaurus",
     "start": "docusaurus start",
-    "build": "docusaurus build",
+    "build": "docusaurus build --no-minify",
```
[the remark this answers is not in the document]
Comment (AngeloCG97) [reply package.json:8]: Minifying the build generate problems for load local font.

File: blog/2021-03-09-Do-you-need-blockchain.mdx (hunk 2 above)
Comment (AngeloCG97) [reply blog/2021-03-09-Do-you-need-blockchain.mdx:42]: @ldrojas

File: src/pages/contact-us.js [hunk 4]
```diff
@@ -1,22 +1,234 @@
...
+  focused:{
+      border: 'solid 1px #5484b3'
+  }
+})((props) => <InputBase {...props} />);
+
 const ContactUs = () => {
   const isMobile = useMediaQuery( {query:'(max-width: 960px)'} )
   const isDesktop = useMediaQuery( {query:'(min-width: 960px)'} )
+  const [resultsSection, setResultsSection] = useState(false)
+
```
Comment (tetogomez) [inline src/pages/contact-us.js:45]: delete blank space

File: src/theme/Navbar.js [hunk 5]
```diff
@@ -159,28 +244,34 @@ const NavbarMenu = ({isMobile, isDesktop}) => {
...
-                      <Grid component="label" container alignItems="center" spacing={1}>
-                        <Grid item>Español</Grid>
-                        <Grid item>
-                          <label className="switch">
-                            <input type="checkbox" checked onChange={() => window.open("https://es.eoscostarica.io/","_self")}/>
-                            <span className="slider round"></span>
-                          </label>
-                        </Grid>
-                        <Grid item>English</Grid>
+                  {/*
```
Comment (tetogomez) [inline src/theme/Navbar.js:257]: please delete commented code

**Review by tetogomez — COMMENTED** [review]
Check comments

File: package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (xavier506) [reply package.json:8]: what do you thinkl about creating a new line :


```
"build": "docusaurus build",
"build-local": "docusaurus build --no-minify",
```
This way the site we publish is minified

**Pull request merged**